### ISSUE TITLE
NVDA announces selection removed before announcing the next list item in focus everytime one moves with ARROWS in font and font style controls in font dialog in MS Word.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3048]
**Reported by sumandogra on 2013-03-04 12:14**
MS Word 2010
NVDA announces "selection removed" everytime on navigating the list items of the font and font style controls in the font dialog. This sometimes is confusing and takes some extra time while moving vastly ;in the list items.
Steps to reproduce:
1.  InMS Word, press CTRL+D to open font dialog.
2. Use UP/DOWN ARROW to move in the Font and font style lists.
3.  NVDA first anounces "selection removed" folowed by the list item in focus then says selected.  
“selection removed” is an extra speech.  This may take some extra time for a user to select the right item.

**Comment by leonardder** [comment]
@sumandogra: Can you still reproduce this in Word 2010? I can't in Word 2016.

**Comment by Adriani90** [comment]
I cannot reproduce it in MS Word 2016 on Windows 10 1809 Update with NVDA 2018.4 Beta 2.
i will test tomorrow on Windows 8.1 in MS Word 2010.

**Comment by Adriani90** [comment]
I cannot reproduce it in MS Word 2010 with NVDA 2017.3 on Windows 8.1.

**Comment by leonardder** [comment]
Closing as works for me
